docs/data-persistence.md

```diff
@@ -5,22 +5,75 @@ Persisting our Redux state to browser storage (IndexedDB) allows us to avoid com
 Redux tree from scratch on each page load and to display cached data in the UI (instead of placeholders)
 while fetching the latest updates from the REST API is still in progress.
 
+Note that the entire Redux state is *not* persisted to the browser. In order to persist state in browser storage the reducer must be wrapped with `withSchemaValidation` as instructed below.
+
 This feature was originally implemented in [#2754](https://github.com/Automattic/wp-calypso/pull/2754).
 
 At a high level, implementing this is straightforward. We subscribe to any Redux store changes, and on change we update
 our browser storage with the new state of the Redux tree. On page load, if we detect stored state in browser storage during
-our initial render, we create our Redux store with that persisted initial state.
+our initial render, we create our Redux store with that persisted initial state. However, significat issues exist that require special solutions:
```

Comment:
Tiny typo!
`significat` should be `significant` :)

Reply:
> Tiny typo!
> `significat` should be `significant` :)

I think you mean a significant typo??!?!  😂